backend/src/schema/resolvers/users.spec.js

```diff
@@ -568,3 +568,330 @@ describe('DeleteUser', () => {
     })
   })
 })
+
+
+
+describe('DeleteUser as Admin', () => {
+  const deleteUserMutation = gql`
+    mutation($id: ID!, $resource: [Deletable]) {
+      DeleteUser(id: $id, resource: $resource) {
+        id
+        name
+        about
+        deleted
+        contributions {
+          id
+          content
+          contentExcerpt
+          deleted
+          comments {
+            id
+            content
+            contentExcerpt
+            deleted
+          }
+        }
+        comments {
+          id
+          content
+          contentExcerpt
+          deleted
+        }
+      }
+    }
+  `
+  beforeEach(async () => {
+    variables = { id: ' u343', resource: [] }
+
+    user = await Factory.build('user', {
+      name: 'My name should be deleted',
+      about: 'along with my about',
+      id: 'u343',
+    })
+    await Factory.build(
+      'user',
+      {
+        id: 'not-my-account',
+      },
+      {
+        email: 'friends-account@example.org',
+      },
+    )
+  })
+
+  describe('unauthenticated', () => {
+    it('throws authorization error', async () => {
+      const { errors } = await mutate({ mutation: deleteUserMutation, variables })
+      expect(errors[0]).toHaveProperty('message', 'Not Authorised!')
+    })
+  })
+
+  describe('authenticated as Admin', () => {
+    beforeEach(async () => {
+      const admin = await Factory.build(
+        'user',
+        {
+          role: 'admin',
+        },
+        {
+          email: 'admin@example.org',
+          password: '1234',
+        },
+      )
+      authenticatedUser = await admin.toJson()
+    })
+ 
+
+    describe('attempting to delete a foreign account by an Admin', () => {
```

Comment:
```suggestion
    describe('deleting a user account', () => {
```
this would then read like `authenticated as Admin deleting a user account`